src/worker/read-wordlist.ts

```diff
@@ -45,3 +45,15 @@ function asNonNegativeInteger(x: unknown): number {
 
   return n;
 }
+
+export async function readManualEntryData(
+  contents: { word: string; count: number | undefined }[]
+): Promise<WordList> {
```

Comment:
Only functions that actually use promises or other the `await` syntax should be marked as `async`:

```suggestion
export function readManualEntryData(
  contents: { word: string; count: number | undefined }[]
): WordList {
```